lib/instrumentation.ex

```diff
@@ -99,17 +101,13 @@ defmodule OpentelemetryAbsinthe.Instrumentation do
     :ok
   end
 
-  defp get_graphql_selections(data) do
-    case data do
-      %{blueprint: %{operations: [_ | _] = operations}} ->
-        operations
-        |> Enum.flat_map(& &1.selections)
-        |> Enum.map(& &1.name)
-        |> Enum.uniq()
-
-      _ ->
-        []
-    end
+  defp get_graphql_selections(%{blueprint: %Blueprint{} = blueprint}) do
+    blueprint
+    |> Blueprint.current_operation()
+    |> Kernel.||(%{})
```

Comment:
Why not

```suggestion
    |> Blueprint.current_operation() || %{}
```

it's just esthetics, but I wondering if there's a reason to use `Kernel.||` instead of inline `||`

Reply:
I remembered this not working, so I just assumed this was invalid syntax or similar, but this comment made me look into it.

This doesn't work because of operator precedence. What happens is basically
```  
defp get_graphql_selections(%{blueprint: %Blueprint{} = blueprint}) do
    Blueprint.current_operation(blueprint) || fallback()
  end

  defp fallback() do
    %{}
    |> Map.get(:selections, [])
    |> Enum.map(& &1.name)
    |> Enum.uniq()
  end
```
it will only ever execute the second part of the pipeline on an empty map.

Reply:
Ok! Interesting and good to know, thanks for looking into it :heart: